### PR TITLE
Pillory / Stocks

### PR DESCRIPTION
**Pull request by Gar-Delin**
What's new?

Pillory/Stocks!
-You can buckle people to them;
-Locks/unlocks by clicking it with the key/keyring. Dungeon and Garrison keys both fit.
-You can not reach the lock while you're buckled;
-As it was suggested, if you have 18+ STR, you can force break free simply by pressing unbuckle. (takes time)

I did not add the recipes, so it's up to the mappers to place them around the map.

### CODE REVIEW

**Comment by Gar-Delin** [comment]
Screenshot_2

**Comment by coffeecoffeecoffee** [comment]
I just want to say these look damn great and the fact that they work with fucking goblins is HILARIOUS.

**Comment by X404ErrorX** [comment]
They look great, maybe can you make them cafteable and work with custom keys/locks too?

**Comment by Gar-Delin** [comment]
> They look great, maybe can you make them cafteable and work with custom keys/locks too?

Maybe later. I want to see how the people will interact with the base version in game for now. (:

**Comment by Hmeister-real** [comment]
> > They look great, maybe can you make them cafteable and work with custom keys/locks too?
> 
> Maybe later. I want to see how the people will interact with the base version in game for now. (:

if you make it craftable, make it use steel ingots, 4 small logs, and make it use a fuck ton of carpentry know how, like expert only

File: modular/code/game/objects/pillory.dm [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	if (!anchored)
+		return FALSE
+
+	if(locked)
+		to_chat(usr, span_warning("Unlock it first!"))
+		return FALSE
+
+	if (!istype(M, /mob/living/carbon/human))
+		to_chat(usr, span_warning("It doesn't look like [M.p_they()] can fit into this properly!"))
+		return FALSE // Can't decapitate non-humans
```
Comment (Elli-Skala) [inline modular/code/game/objects/pillory.dm:80]: copy paste comment error

Comment (Gar-Delin) [reply modular/code/game/objects/pillory.dm]: Just one missed comments message... 🙄

Fixed.

**Comment by zakanater19** [comment]
Nice

**Comment by Gr33d-y** [comment]
it seems the pillory can not be unlatched.

**Comment by Useroth** [comment]
Merged into #417 and expanded slightly

**Pull request closed without merging**